src/lib.rs

```diff
@@ -365,14 +393,14 @@ unsafe fn list_push_gen<T, W: Pod>(meta: &mut BufSlice<W>, data: &mut BufSlice<W
 
     let mut ptr = data.as_mut_ptr() as *mut T;
     let mut clr = PanicState(ptr, 0, reset_slot, reset_value);
-    for i in 0 .. count {
+    for i in 0..count {
         let val = gen(i);
         ptr::write(ptr, val);
         ptr = ptr.offset(1);
         clr.1 += 1;
     }
-    clr.0 = ptr::null_mut();    // Prevent drops and prevent reset
-    // Save the length once everything has been written
+    clr.0 = ptr::null_mut(); // Prevent drops and prevent reset
+                             // Save the length once everything has been written
```

Comment:
nitpick: this probably needed a newline, because these two comments aren't linked - but rustfmt seems to have assumed they are.

Reply:
Sure! that's now fixed